src/components/subscriptions/licenses/LicenseManagementModals/tests/LicenseManagementRevokeModal.test.jsx

```diff
@@ -158,4 +159,73 @@ describe('<LicenseManagementRevokeModal />', () => {
       expect(screen.getByRole('alert')).toBeTruthy();
     });
   });
+
+  describe('calls the correct revoke endpoint', () => {
+    beforeEach(() => {
+      const mockPromiseResolve = Promise.resolve({ data: {} });
+      LicenseManagerApiService.licenseRevokeAll.mockReturnValue(mockPromiseResolve);
+      LicenseManagerApiService.licenseBulkRevoke.mockReturnValue(mockPromiseResolve);
```

Comment:
nit: this works, but could probably use the helper function `mockResolvedValue({ data: {} })` as well

Reply:
agree should be using mockResolvedValue here